src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodNameCheck.java

```diff
@@ -83,6 +83,17 @@
  * <pre>
  * &lt;module name="MethodName"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * class MyClass {
+ *   public void first_Method1() {} // OK
+ *   protected void second_Method2() {} // OK
+ *   private void Third_Method3() {} // violation, name 'ThirdMethod' must match
```

Comment:
These names doesn't align to default `MethodNameCheck`.
Since underscore is not allowed in default config.
Use should only use CamelCasing for naming methods here.

Reply:
Resolved